10up/wp-scaffold
# #187 Added @10up/babel-preset-default to dev dependencies.

### Description of the Change

Adds @10up/babel-preset-default to dev dependencies

Closes #187 

### How to test the Change

1. Note and test the repeatable steps on #187 to see the error on the `trunk` branch
2. Switch to this branch and run: `npm install; npm run build`
3. Build should complete successfully


### Changelog Entry
> Fixed - Missing dependency


### Credits
Props @jamesmorrison


### Checklist:
